packages/sandbox/src/helpers/postMessageRelayHelpers.ts

```diff
@@ -188,6 +189,7 @@ export type IncomingEmbedMessage =
       name: typeof INTROSPECTION_QUERY_WITH_HEADERS;
       introspectionRequestBody: string;
       introspectionRequestHeaders: Record<string, string>;
+      includeCookies: boolean;
```

Comment:
e[mbedded Sandbox & embedded Explorer still use service workers](https://github.com/mdg-private/studio-ui/blob/main/packages/studio/src/serviceWorkerSetup/serviceWorkerRegistration.ts#L123-L126), so we could have older versions of studio using this package. Can we add a comment here & in the other explorer file to explain that & make these optional because we can't guarentee studio will send them? 

Maybe in the comment we can add a TODO to delete in a few months?